src/main/java/io/gravitee/policy/transformheaders/TransformHeadersPolicy.java

```diff
@@ -194,7 +194,7 @@ void transform(HttpHeaders httpHeaders, ExecutionContext executionContext) {
             httpHeaders
                 .names()
                 .forEach(headerName -> {
-                    if (!transformHeadersPolicyConfiguration.getWhitelistHeaders().contains(headerName)) {
+                    if (transformHeadersPolicyConfiguration.getWhitelistHeaders().stream().noneMatch(headerName::equalsIgnoreCase)) {
```

Comment:
I'm wondering if we can find a proper structure for the whitelist headers that would be natively case-insensitive (or just lowercase the list in the constructor) so we can use `whitelistHeaders.contains(headerName.toLowerCase()) ` instead of iterating?

Reply:
As discussed together, seems ok to leave it like this 👍